src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -117,6 +117,14 @@ public function getPath()
         return $this->path;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getBundleDir(): string
+    {
+        return $this->getPath();
```

Comment:
if the behavior is exactly the same, we don't need a new method!

Reply:
Indeed, it shouldn't be the default value for the new structure. I've updated it, check it out again.